web/src/components/molecules/Content/List/index.tsx

```diff
@@ -20,11 +21,16 @@ export type Props = {
   contentTableColumns?: ProColumns<ContentTableField>[];
   modelsMenu: React.ReactNode;
   selectedItem: Item | undefined;
+  selection: {
+    selectedRowKeys: Key[];
```

Comment:
Is selectedRowKeys sometimes a string AND sometimes a number? Usually in this situation we either use, for example field IDs, which are strings, or the index while we map through another array, which is always a number.
If possible, just use a simple type instead of the Key type from React.

Reply:
@KaWaite , it is used like this in multiple places now, we are using `Key` following and docs, but I think it can be removed.